PersonalTwitterFeed/src/personaltwitterfeed/PersonalTwitterFeed.java

```diff
@@ -20,7 +21,16 @@
     /**
      * @param args the command line arguments
      */
+  
+    public static String Date(){
```

Comment:
The name of your method is more consistent with its function compared to my code.